Dockerfile

```diff
@@ -12,28 +12,26 @@ COPY . /ext-scylladb
 
 WORKDIR /ext-scylladb
 
-RUN php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"  \
```

Comment:
I dont thinks it's good idea to remove composer and image update, since this image will not be a production image, and more of a testing grounds, `composer` will be required for Pest/PhpUnit to install dependencies

Reply:
composer is installed via install-php-extensions.

Reply:
now I've seen it, but i would still use the official way of installing it

Reply:
`install-php-extensions` uses the official source with signature verification: https://github.com/mlocati/docker-php-extension-installer/blob/19fef35c08a539686e445cd242152e86d5d47a31/install-php-extensions#L2103
If you don't like it, why are you using `install-php-extensions` for extensions?

Reply:
> If you don't like it, why are you using install-php-extensions for extensions?

Old habits, I guess.